docs/api-reference/mesh-layers/simple-mesh-layer.md

```diff
@@ -101,9 +101,10 @@ The geometry to render for each data object. One of:
 - An URL to a mesh description file in a format supported by [loaders.gl](https://loaders.gl/docs/specifications/category-mesh). The appropriate loader will have to be registered via the loaders.gl `registerLoaders` function for this usage.
 - A luma.gl [Geometry](https://luma.gl/docs/api-reference/engine/geometry) instance
 - An object containing the following fields:
-  + `positions` (Float32Array) - 3d vertex offset from the object center, in meters
-  + `normals` (Float32Array) - 3d normals
-  + `texCoords` (Float32Array) - 2d texture coordinates
+  + `positions` (Float32Array) - 3d vertex offset from the object center, in meters (required)
```

Comment:
The convention in our docs is `Float32Array, optional`

Reply:
Done